src/VisualStudio/Core/Test/Options/BasicEditorConfigGeneratorTests.vb

```diff
@@ -0,0 +1,162 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Text
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests
+    <[UseExportProvider]>
+    Public Class BasicEditorConfigGeneratorTests
+        Inherits TestBase
+
+        <ConditionalWpfFact(GetType(IsEnglishLocal))>
+        Public Sub TestEditorConfigGeneratorDefault()
+            Using workspace = TestWorkspace.CreateVisualBasic("")
+                Dim expectedText = "# Comment the line below if you want to inherit parent .editorconfig settings
```

Comment:
consider "from higher directories" instead of "parent".  
consider: "remove the line below"